app/src/main/java/com/keylesspalace/tusky/BaseActivity.java

```diff
@@ -93,6 +97,18 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         requesters = new HashMap<>();
     }
 
+    @Override
+    protected void attachBaseContext(Context newBase) {
+        SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(newBase);
+
+        float uiScaleRatio = preferences.getFloat(PrefKeys.UI_SCALE_RATIO, 100F);
+        Context context = FontScaleContextWrapper.Companion.wrap(
```

Comment:
When you annotate a method with `@JvmStatic` you can call it from Java without `Companion`